fe/fe-core/src/main/java/com/starrocks/analysis/RevokeRoleStmt.java

```diff
@@ -0,0 +1,19 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.analysis;
+
+// REVOKE Role 'role' FROM 'user'
+// share the same parameter and check logic with GrantRoleStmt
+public class RevokeRoleStmt extends GrantRoleStmt {
```

Comment:
RevokeRole should be at the same level with GrantRole.

Reply:
What does `level` mean? Are you saying that RevokeRoleStmt cannot be a subclass of GrantRoleStmt? Since the inner logic between those two statements is basically the same, there seem to be 3 ways to implement this.

1. Copy the code from GrantRoleStmt to RevokeRoleStmt, and make some minor changes. It's a common way but I think it's ugly.
2. Abstract the logic to one class, saying `GrantRevokeRoleStmt`, add a flag/enum parameter to suggest whether is `granted` or `revoke`. It's weird to use and brings out a lot of if/else branches, which make it harder to understand.
3. Using inheritance to reuse the common logic, rewriting the difference by implementing corresponding interfaces. It still seems a bit strange when you jump to the definition of the class at first sight, but not to the one who merely uses this class because it acts as a normal class of Statement.

Considering these, as well as the fact that I really suffered a lot when I try to read the code the first time, I chose the 3rd solution because I think it is easier to maintain.